sounds/samples/README.md

```diff
@@ -3,6 +3,8 @@ How to add a new sample sound
 
 * Find a sample under a free/libre open-source (FOSS) license.
 
+* The sample has to have only one note (pitch) through out the audio and the volume has to be 50 . The longer the sample, the better it is.
```

Comment:
I don't understand what you are saying here. Also, a long sample length is not always appropriate or better.

Reply:
About the notes, it's mentioned here [link](https://github.com/sugarlabs/musicblocks/pull/1485). And in the issue #1468 pikurasa mentioned about including about longer tones.

Reply:
I agree with his first three points, but not the last one. It really depends on the instrument. Maybe you can use the "check box" approach he used in the comment. I think it is clearer than the wording you used.

Reply:
I will use checkbox and remove the longer tones part for now.